system/assets/js/admin/presidecore/formFields.js

```diff
@@ -105,4 +105,10 @@
     	}
     });
 
+
+    $('.icon-picker-fontawesome').iconpicker({
+        title: 'Using Font Awesome',
```

Comment:
This can/should use:

    title : i18n.translateResource( 'cms:font.awesome.picker.title' )

You can then add corresponding i18n .properties entry.